specs/corla.pvs

```diff
@@ -0,0 +1,923 @@
+%%% model of the Colorado Risk Limiting Audit (CORLA) tool.
```

Comment:
After some discussion of a variety options for continuing the review of corla.pvs, @ranweiler and I figure the easiest approach, which provides a convenient interface for commenting on any individual line in the file, is to just continue adding review comments in this original PR for the code. Naturally, we can't use the "request changes" feature of the review tool, since the changes have already been merged, but that's no loss.